setup.py

```diff
@@ -48,7 +48,8 @@
         'Sphinx>=1.4.2',
     ],
     'badges': [
-        'CairoSVG>=1.0.20',
+        # CairoSVG 2.0.0 only supports Python 3
+        'CairoSVG>=1.0.20,<2.0.0',
```

Comment:
[Environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) should be supported since [pip 8.1.2](https://github.com/pypa/pip/issues/3624).

We should be able to do something like:
```python
'CairoSVG>=1.0.20,<2.0.0;python_version<"3.0"', # CairoSVG 2.0.0+ only supports Python 3
'CairoSVG>=1.0.20;python_version>="3.0"',
```